src/components/Anatomy.jsx

```diff
@@ -288,39 +307,44 @@ export const AnatomyLevel1 = () => {
       messageRef.current.rotation.x += 0.01;
     });
     if (gameOver) {
-      return <group ref={messageRef}><Center rotation={[-0.5, -0.2, 0]}>
+      return <group ref={messageRef}><Suspense><Center rotation={[-0.5, -0.2, 0]}>
         <Text3D
+          url="https://api.fontsource.org/v1/fonts/lora/latin-600-italic.ttf"
           curveSegments={32}
           bevelEnabled
           bevelSize={0.04}
           bevelThickness={0.1}
           height={0.5}
-          lineHeight={0.5}
-          letterSpacing={-0.06}
+          lineHeight={1.0}
+          letterSpacing={0.06}
           size={1.5}
-          font="/Inter_Bold.json">
+          //font="/Inter_Bold.json"
```

Comment:
I am not really sure why you are changing the Text3D? 
I think it should work now without any changes required after I added the Inter_Bold.json font file yesterday to main.
Am I missing something here?

Reply:
So, I reverted the Anatomy.jsx file to the main branch version.